src/driver/resnet50.cpp

```diff
@@ -27,1031 +27,1564 @@
+    auto x_main_module_11 = mmain->add_literal(
+        migraphx::generate_literal(migraphx::shape{migraphx::shape::float_type, {512}}, 10));
+    auto x_main_module_12 = mmain->add_literal(migraphx::abs(
+        migraphx::generate_literal(migraphx::shape{migraphx::shape::float_type, {512}}, 11)));
     auto x_main_module_13 = mmain->add_literal(migraphx::generate_literal(
         migraphx::shape{migraphx::shape::float_type, {512, 512, 3, 3}}, 12));
-    auto x_main_module_14 = mmain->add_literal(
-        migraphx::generate_literal(migraphx::shape{migraphx::shape::float_type, {512}}, 13));
-    auto x_main_module_15 = mmain->add_literal(migraphx::generate_literal(
-        migraphx::shape{migraphx::shape::float_type, {512, 2048, 1, 1}}, 14));
+    auto x_main_module_14 = mmain->add_literal(migraphx::abs(
+        migraphx::generate_literal(migraphx::shape{migraphx::shape::float_type, {512}}, 13)));
+    auto x_main_module_15 = mmain->add_literal(
+        migraphx::generate_literal(migraphx::shape{migraphx::shape::float_type, {512}}, 14));
     auto x_main_module_16 = mmain->add_literal(
-        migraphx::generate_literal(migraphx::shape{migraphx::shape::float_type, {2048}}, 15));
-    auto x_main_module_17 = mmain->add_literal(migraphx::generate_literal(
-        migraphx::shape{migraphx::shape::float_type, {2048, 1024, 1, 1}}, 16));
-    auto x_main_module_18 = mmain->add_literal(
-        migraphx::generate_literal(migraphx::shape{migraphx::shape::float_type, {2048}}, 17));
-    auto x_main_module_19 = mmain->add_literal(migraphx::generate_literal(
-        migraphx::shape{migraphx::shape::float_type, {2048, 512, 1, 1}}, 18));
+        migraphx::generate_literal(migraphx::shape{migraphx::shape::float_type, {512}}, 15));
+    auto x_main_module_17 = mmain->add_literal(migraphx::abs(
+        migraphx::generate_literal(migraphx::shape{migraphx::shape::float_type, {512}}, 16)));
+    auto x_main_module_18 = mmain->add_literal(migraphx::generate_literal(
+        migraphx::shape{migraphx::shape::float_type, {512, 2048, 1, 1}}, 17));
+    auto x_main_module_19 = mmain->add_literal(migraphx::abs(
+        migraphx::generate_literal(migraphx::shape{migraphx::shape::float_type, {2048}}, 18)));
     auto x_main_module_20 = mmain->add_literal(
-        migraphx::generate_literal(migraphx::shape{migraphx::shape::float_type, {512}}, 19));
-    auto x_main_module_21 = mmain->add_literal(migraphx::generate_literal(
-        migraphx::shape{migraphx::shape::float_type, {512, 512, 3, 3}}, 20));
-    auto x_main_module_22 = mmain->add_literal(
-        migraphx::generate_literal(migraphx::shape{migraphx::shape::float_type, {512}}, 21));
+        migraphx::generate_literal(migraphx::shape{migraphx::shape::float_type, {2048}}, 19));
+    auto x_main_module_21 = mmain->add_literal(
+        migraphx::generate_literal(migraphx::shape{migraphx::shape::float_type, {2048}}, 20));
+    auto x_main_module_22 = mmain->add_literal(migraphx::abs(
```

Comment:
Why the sudden use of migraphx::abs() between the previous iteration?

Reply:
These literal need to be positive numbers.

Reply:
Oh okay.